extensions/cloudfiles-extensions/pom.xml

```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0"?>
+
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>io.druid.extensions</groupId>
+	<artifactId>druid-cloudfiles-extensions</artifactId>
+	<name>druid-cloudfiles-extensions</name>
+	<description>druid-cloudfiles-extensions</description>
+
+	<parent>
+		<groupId>io.druid</groupId>
+		<artifactId>druid</artifactId>
+		<version>0.8.0</version>
```

Comment:
This will need to be updated as the PR moves along. Current master is 0.8.2-SNAPSHOT